cmd/influx/bucket.go

```diff
@@ -25,9 +26,8 @@ func bucketF(cmd *cobra.Command, args []string) {
 
 // BucketCreateFlags define the Create Command
 type BucketCreateFlags struct {
-	name      string
-	orgID     string
-	org       string
+	name string
+	organization
```

Comment:
I'd recommend not embedding the organization type, although, all this will likely get torched when a testable design comes in 🤔

Reply:
totally agree, I wanted this to be easy to revert once we converge on a pattern we're implementing across the board.